### PR TITLE
fix: webwallet trpc link for popups

### PR DESCRIPTION
**Pull request by bluecco**
### Issue / feature description

`Quick description of the issue or the feature that is implemented in the pr ie: add new transitions between screens`

### Changes

`quick list regarding the related changes within the pr`

ie:

- add new connector
- fix injected connector
- ...

### Checklist

- [ ] Rebased to the last commit of the target branch (or merged)
- [ ] Code self-reviewed
- [ ] Code self-tested
- [ ] Tests updated (if needed)
- [ ] All tests are passing locally

Please keep your pull request as small as possible. If you need to make multiple changes, please create separate pull requests for each change. Create a draft pull request if you need early feedback. This will mark the pull request as a work in progress and prevent it from being reviewed or merged until you are ready.

Please move drafts to the ready for review (regular PR) when you are ready to start the review process and other developers will pick it up from there.

### CODE REVIEW

**Review by Cussone — APPROVED** [review]
I don't understand what is different between this and the previous approach, maybe an answer to some of the questions I asked answers that

File: src/connectors/webwallet/helpers/trpc.ts [hunk 1]
```diff
@@ -195,29 +225,35 @@ export const trpcProxyClient = ({
...
-              }
-            })()
+            try {
+              let popup: Window | null = null
+              const button = document.createElement("button")
+              button.style.position = "fixed"
+              button.style.top = "-999px"
+              button.style.left = "-999px"
+              button.style.opacity = "0"
+              button.style.pointerEvents = "none" // prevent click event
```
[the remark this answers is not in the document]
Comment (bluecco) [reply src/connectors/webwallet/helpers/trpc.ts:235]: did some searching and it seems it's more reliable

- Browser Security: Some browsers treat elements with display: none as "not truly interactive" from a security perspective, which can affect how they handle popup permissions
- DOM Interaction: While display: none removes the element from the layout flow completely, position: fixed keeps it in the DOM's interactive layer
- Mobile Browsers: Particularly on mobile browsers, clicks on display: none elements might be ignored or treated differently than clicks on positioned-but-invisible elements

File: src/connectors/webwallet/helpers/trpc.ts [hunk 2]
```diff
@@ -53,6 +53,36 @@ export const setPopupOptions = ({
...
+
+    if (!popup) {
+      throw new Error("Popup blocked by browser")
+    }
+
+    this.currentPopup = popup
+
+    // Monitor popup state
+    const checkClosed = setInterval(() => {
+      if (popup.closed) {
```
[the remark this answers is not in the document]
Comment (bluecco) [reply src/connectors/webwallet/helpers/trpc.ts:76]: could be an edge case, but theoretically `this.currentPopup !== popup` should not happen in this case
popup is always the new one

thinking if this approach could be more reliable but it would requires some tests (using `beta` branch)

File: src/connectors/webwallet/helpers/trpc.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Cussone) [reply src/connectors/webwallet/helpers/trpc.ts:235]: Interesting. I saw something like this coming, legit 👍🏼
